website/source/docs/vault-integration/index.html.md

```diff
@@ -0,0 +1,351 @@
+  `nomad-cluster`. If a different name is chosen, replace the token role in the
+  above policy.
+
+*   `orphan` - Specifies whether tokens created against this token role will be
+  orphaned and have no parents. Nomad does not enforce the value of this field
+  but understanding the implications of each value is important.
+
+    If set to false, all tokens will be revoked when the Vault token given to
+  Nomad expires. This makes it easy to revoke all tokens generated by Nomad but
+  forces all Nomad servers to use the same Vault token, even through upgrades of
+  Nomad servers. If the Vault token that was given to Nomad and used to generate
+  a tasks token expires, the token used by the task will also be revoked which
+  is not ideal.
+
+    When set to true, the tokens generated for tasks will not be revoked when
+  Nomad's token is revoked. However Nomad will still revoke tokens when the
+  allocation is no longer running, minimizing the lifetime of any task's token.
+  With orphaned enabled, each Nomad server may also use a unique Vault token,
+  making bootstrapping and upgrading simpler. As such, **setting `orphan = true`
+  is the recommended setting**.
+
+* `period` - Specifies the length the TTL is extended by each renewal in
+  seconds. It is suggested to set this value on the order of magnitude of 3 days
+  (259200 seconds) to avoid a large renewal request rate to Vault. **Must be set
+  to a positive value**.
+
+* `renewable` - Specifies whether created tokens are renewable. **Must be set to
+  `true`**. This allows Nomad to renew tokens for tasks.
+
+The above [`nomad-cluster` token role](/data/vault/nomad-cluster-role.json) is
+available for download. Below is an example of writing this role to Vault:
+
+```
+# Download the token role
+$ curl https://nomadproject.io/data/vault/nomad-cluster-role.json -O -s -L
+
+# Create the token role with Vault
+$ vault write /auth/token/roles/nomad-cluster @nomad-cluster-role.json
+```
+
+
+#### Example Configuration
+
+To make getting started easy, the basic [`nomad-server`
+policy](/data/vault/nomad-server-policy.hcl) and
+[`nomad-cluster` role](/data/vault/nomad-cluster-role.json) described above are
+available for download.
+
+The below example assumes Vault is accessible, unsealed and the operator has
+appropriate permissions.
+
+```shell
+# Download the policy and token role
+$ curl https://nomadproject.io/data/vault/nomad-server-policy.hcl -O -s -L
+$ curl https://nomadproject.io/data/vault/nomad-cluster-role.json -O -s -L
+
+# Write the policy to Vault
+$ vault policy write nomad-server nomad-server-policy.hcl
+
+# Create the token role with Vault
+$ vault write /auth/token/roles/nomad-cluster @nomad-cluster-role.json
+```
+
+#### Retrieving the Token Role based Token
+
+After the token role is created, a token suitable for the Nomad servers may be
+retrieved by issuing the following Vault command:
+
+```
+$ vault token create -policy nomad-server -period 72h -orphan
```

Comment:
If you are generating a token based on a defined role, the command should be:
`vault token create -role=nomad-cluster`

Otherwise, the previous step to create a role becomes meaningless.